addons/mail/static/src/models/partner/partner.js

```diff
@@ -282,8 +282,8 @@ function factory(dependencies) {
                     return 1;
                 }
                 if (thread && thread.model === 'mail.channel') {
-                    const isAMember = thread.members.includes(a);
-                    const isBMember = thread.members.includes(b);
+                    const isAMember = thread.threadMembers.includes(a);
+                    const isBMember = thread.threadMembers.includes(b);
```

Comment:
similar to everywhere else, whenever code checked for "members" before should now check for "members.partner"